CHANGELOG-3.8.md

```diff
@@ -58,6 +58,60 @@ These are the changes since the Ice 3.7.10 release in [CHANGELOG-3.7.md](./CHANG
   (the unit is seconds). You can also override this value for a specific object adapter with the configuration
   property `AdapterName.Connection.CloseTimeout`.
 
+- Simplify proxy creation.
+  You can now create a typed proxy directly from a communicator and a string in all languages. For example:
+
+  ```cpp
+  // C++
+  GreeterPrx greeter{communicator, "greeter:tcp -h localhost -p 4061"};
+  ```
+
+  ```csharp
+  // C#
+  var greeter = GreeterPrxHelper.createProxy(communicator, "greeter:tcp -h localhost -p 4061");
+  ```
+
+  ```java
+  // Java
+  var greeter = GreeterPrx.createProxy(communicator, "greeter:tcp -h localhost -p 4061");
+  ```
+
```

Comment:
I don't know what it looks like in JavaScript.

Reply:
`const greater = new GreeterPrx(communicator, "...")`